nncf/tensorflow/quantization/algorithm.py

```diff
@@ -464,8 +469,7 @@ def _get_quantizer_setup(self, model: tf.keras.Model) -> TFQuantizationSetup:
                         weight_def.weight_attr_name)
                     self._op_names.append(op_name)
 
-                    half_range = self._get_half_range(qconfig)
-                    applied_saturation_fix = applied_saturation_fix or half_range
+                    half_range, applied_saturation_fix = self._get_half_range(qconfig, applied_saturation_fix)
```

Comment:
I'm not sure, but it seems that the quantizer setup is not required to be stored in the order of traversal of the graph. Could you propose how this can be done more reliably? cc' @vshampor

Reply:
Fixed, thanks!